flink-libraries/flink-table/src/test/java/org/apache/flink/table/api/java/utils/UserDefinedAggFunctions.java

```diff
@@ -59,7 +59,7 @@ public boolean requiresOver() {
 	/**
 	 * Accumulator for WeightedAvg.
 	 */
-	public static class WeightedAvgAccum extends Tuple2<Long, Integer> {
```

Comment:
TypeExtractor will recognize `WeightedAvgAccum` as a `TupleType` which will ignore the `sum` and `count` fields when de/serialize. So I changed it as a POJO class. This should not be included in this PR. But I fixed it when I find it.